commands/create/kubeconfig/command.go

```diff
@@ -552,18 +591,20 @@ func createKubeconfig(ctx context.Context, args Arguments) (createKubeconfigResu
 			result.contextName = "giantswarm-" + clusterID
 		}
 
-		// edit kubectl config
-		if err := util.KubectlSetCluster(clusterID, result.apiEndpoint, result.caCertPath); err != nil {
-			return result, microerror.Mask(util.CouldNotSetKubectlClusterError)
-		}
-		if err := util.KubectlSetCredentials(clusterID, result.clientKeyPath, result.clientCertPath); err != nil {
-			return result, microerror.Mask(util.CouldNotSetKubectlCredentialsError)
-		}
-		if err := util.KubectlSetContext(result.contextName, clusterID); err != nil {
-			return result, microerror.Mask(util.CouldNotSetKubectlContextError)
-		}
-		if err := util.KubectlUseContext(result.contextName); err != nil {
-			return result, microerror.Mask(util.CouldNotUseKubectlContextError)
+		if !args.useKubie {
+			// edit kubectl config
+			if err := util.KubectlSetCluster(clusterID, result.apiEndpoint, result.caCertPath); err != nil {
+				return result, microerror.Mask(util.CouldNotSetKubectlClusterError)
+			}
+			if err := util.KubectlSetCredentials(clusterID, result.clientKeyPath, result.clientCertPath); err != nil {
+				return result, microerror.Mask(util.CouldNotSetKubectlCredentialsError)
+			}
+			if err := util.KubectlSetContext(result.contextName, clusterID); err != nil {
+				return result, microerror.Mask(util.CouldNotSetKubectlContextError)
+			}
```

Comment:
This means that when `--kubie` is applied, the necessary entries in the user's kubectl config file won't be created. This causes https://github.com/giantswarm/giantswarm/issues/15957

Reply:
I'll provide a fix

Reply:
https://github.com/giantswarm/gsctl/pull/610

Reply:
Thanks for fixing ❤️